### DISCUSSION TITLE
How to implement electron-vite in electron-react-boilerplate ?

### DISCUSSION THREAD

**Discussion by DynMatrix** [alex8088/electron-vite#409] [General]
[electron-react-boilerplate ](https://github.com/electron-react-boilerplate/electron-react-boilerplate) Electron React Boilerplate uses [Electron](https://electron.atom.io/), [React](https://facebook.github.io/react/), [React Router](https://github.com/reactjs/react-router), [Webpack](https://webpack.js.org/) and [React Fast Refresh](https://www.npmjs.com/package/react-refresh). 

I am using electron react boilerplate but unable to protect code plan to implement electron-vite. But electron-react-boilerplate is tightly coupled with webpack and erb . From where to start electron-vite to integrate in electron-react-boilerplate.

**Comment by alex8088** [comment]
https://electron-vite.org/guide/#scaffolding-your-first-electron-vite-project
